M4L_Device/m4l_bridge.js

```
@@ -0,0 +1,1750 @@
+                break;
+            case "remove":
+                sampleApi.call("remove_slice", sliceTime);
+                sendResult({ action: "remove", slice_time: sliceTime }, requestId);
+                break;
+            case "clear":
+                sampleApi.call("clear_slices");
+                sendResult({ action: "clear" }, requestId);
+                break;
+            case "move":
+                var newTime = parseFloat(args[4]);
+                sampleApi.call("move_slice", sliceTime, newTime);
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)